client/reader/following-stream/_style.scss

```diff
@@ -373,3 +373,7 @@
 	font-size: 12px;
 	line-height: 1;
 }
+
+.reader-stream-notice.notice.is-info {
```

Comment:
I think we should perhaps create an `update` status or similar for notices that is orange instead.